plugins/main/README.md

```diff
@@ -26,7 +26,6 @@ the Wazuh Indexer. The plugin provides the following capabilities:
   - Threat Detection and Response
     - Vulnerabilities: Discover what applications in your environment are affected by well-known vulnerabilities.
     - MITRE ATT&CK: Explore security alerts mapped to adversary tactics and techniques for better threat understanding.
-    - VirusTotal: Alerts resulting from VirusTotal analysis of suspicious files via an integration with their API.
```

Comment:
**nitpick**: the Malware Detection application can display data about VirusTotal, so this should not be removed